deployments/gcp-uscentral1b/config/common.yaml

```diff
@@ -90,6 +94,40 @@ pangeo:
           name: custom-templates
           subPath: "pangeo-custom-jupyterhub-templates/extra-assets"
 
+  dask-gateway:
+    gateway:
+      extraConfig:
+        optionHandler: |
+          from dask_gateway_server.options import Options, Integer, Float, String
+          def cluster_options(user):
+             def option_handler(options):
+                 if ":" not in options.image:
+                     raise ValueError("When specifying an image you must also provide a tag")
+                 extra_annotations = {
+                     "hub.jupyter.org/username": user.name,
+                     "prometheus.io/scrape": "true"
+                 }
+                 extra_labels = {
+                     "hub.jupyter.org/username": user.name,
```

Comment:
This lets us (in theory) capture per-user metrics. When user `taugspurger` creates a pod, it'll get the label `hub.jupyter.org/username": "taugspurger"`. When prometheus captures metrics, labels are included.